packages/itwinui-css/src/tile/tile.scss

```diff
@@ -5,6 +5,9 @@
 @import '../utils/index';
 
 @mixin iui-tile {
+  --_iui-tile-title-text-color: var(--iui-color-foreground-2);
+  --_iui-tile-body-text-color: var(--iui-color-foreground-4);
```

Comment:
Is the `_` prefix on the variable name a convention rule to follow when creating css variables in a specific component's scss file? And without `_` means the variable is from the variables package?

Reply:
You may find some instances where we are not following this pattern.  We introduced this pattern a little bit after we started switching to CSS variables.  We should update areas that are not adhering to this pattern.